js/controller.js

```diff
@@ -149,6 +149,16 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
         delayedContentData: null
       },
 
+      scrubberBar: {
+        isHovered: false
```

Comment:
Would it make more sense to call this isHovering?

Reply:
Will change